MIGRATING.md

```diff
@@ -2,6 +2,94 @@
+ncellationToken)`
+- `ActivateFactorAsync(Okta.Sdk.IVerifyFactorRequest verifyFactorRequest,String userId,String factorId,CancellationToken cancellationToken)`
+<br />&mdash;Renamed with new signature `ActivateFactorAsync(Okta.Sdk.IActivateFactorRequest body,String userId,String factorId,CancellationToken cancellationToken)`
+
+**Okta.Sdk.UsersClient**
+- `ListUsers(String q,String after,Int32 limit,String filter,String format,String search,String expand)`
+<br />&mdash; Signature changed `ListUsers(String q,String after,Int32 limit,String filter,String search,String sortBy,String sortOrder)`
+- `CreateUserAsync(Okta.Sdk.IUser user,Boolean activate,Boolean provider,Okta.Sdk.UserNextLogin nextLogin,CancellationToken cancellationToken)`
+<br />&mdash; Signature changed `CreateUserAsync(Okta.Sdk.ICreateUserRequest body,Boolean activate,Boolean provider,Okta.Sdk.UserNextLogin nextLogin,CancellationToken cancellationToken)`
+- `ListAppLinks(String userId,Boolean showAll)`
+<br />&mdash; Signature changed `ListAppLinks(String userId)`
+- `ListUserGroups(String userId,String after,Int32 limit)`
+<br />&mdash; Signature changed `ListUserGroups(String userId)`
+- `ExpirePasswordAsync(String userId,Boolean tempPassword,CancellationToken cancellationToken)`
+<br />&mdash; Signature changed `ExpirePasswordAsync(String userId,CancellationToken cancellationToken)` 
+- `ResetAllFactorsAsync(String userId,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `ResetFactorsAsync(String userId,CancellationToken cancellationToken)`
+- `ResetPasswordAsync(String userId,Okta.Sdk.AuthenticationProviderType provider,Boolean sendEmail,CancellationToken cancellationToken)`
+<br />&mdash; Removed; instead use any of the following:
+  - `ForgotPasswordGenerateOneTimeTokenAsync(String userId,Boolean sendEmail,CancellationToken cancellationToken)`
+  - `ForgotPasswordSetNewPasswordAsync(Okta.Sdk.IUserCredentials user,String userId,Boolean sendEmail,CancellationToken cancellationToken)`
+  - `ExpirePasswordAsync(String userId,CancellationToken cancellationToken)`
+  - `ExpirePasswordAndGetTemporaryPasswordAsync(String userId,CancellationToken cancellationToken)`
+- `ListAssignedRoles(String userId,String expand)`
+<br />&mdash; Renamed `ListAssignedRolesForUser(String userId,String expand)`
+- `EndAllUserSessionsAsync(String userId,Boolean oauthTokens,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `ClearUserSessionsAsync(String userId,Boolean oauthTokens,CancellationToken cancellationToken)`
+
+### New Okta Clients
+The following is a list of context specific clients that are new in version 3.0.  Instances of each are available as properties of an OktaClient instance where the name of the property is the name of the type with the "Client" suffix removed.
+
+- `Okta.Sdk.AuthorizationServersClient`, see [Authorization Servers](https://developer.okta.com/docs/reference/api/authorization-servers/).
```

Comment:
```suggestion
- `Okta.Sdk.AuthorizationServersClient`, see [Authorization Servers API](https://developer.okta.com/docs/reference/api/authorization-servers/).
```

Reply:
I lean toward agreeing with you.  However, the title here should match the title in the target.  The title at the target url (https://developer.okta.com/docs/reference/api/authorization-servers/) is "Authorization Servers".  All the others are suffixed by "API".  For consistency it should be changed in both places; I'll change it here and submit a PR in the docs repo.

Reply:
Updated here and submitted PR to update docs: https://github.com/okta/okta-developer-docs/pull/1277